<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>about</class>
 <widget class="QDialog" name="about">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Word count: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="word_count_line"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Maximum word length: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="max_word_length_line"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="max_word_line"/>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Maximum matches: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="max_matches_line"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QTextEdit" name="matches_textEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
